### PR TITLE
Bump kube dependencies to v1.25.2

### PR DESCRIPTION
**Pull request by jonesbr17**
Using latest available releases

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -129,75 +126,72 @@ require (
...
+	google.golang.org/grpc v1.47.0 // indirect
+	google.golang.org/protobuf v1.28.0 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/klog v1.0.0 // indirect
 	k8s.io/kube-aggregator v0.24.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42 // indirect
+	k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1 // indirect
```
Comment (rtheis) [inline go.mod:147]: Please bump `k8s.io/kube-aggregator` to 1.25 as well.

**Review by rtheis — CHANGES_REQUESTED** [review]
Let's use 1.25.2 now that it was just released.

File: go.mod (hunk 1 above)
Comment (jonesbr17) [reply go.mod:147]: This is an indirect dependency. We override this further down in the file: `k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.25.2`

**Comment by jonesbr17** [comment]
Updated to use Kube 1.25.2 dependencies

**Review by rtheis — APPROVED** [review]
/lgtm
/approved

**Comment by rtheis** [comment]
@jonesbr17 it appears that you will need to manually cherry-pick this to the release-4.12 branch.

**Comment by jonesbr17** [comment]
Manual cherry-pick created for 4.12 https://github.com/openshift/ibm-roks-toolkit/pull/650

**Pull request merged**